tests/graph/test_centrality.py

```diff
@@ -230,3 +230,46 @@ def test_custom_graph_unnormalized(self):
         expected = {0: 9, 1: 9, 2: 12, 3: 15, 4: 11, 5: 14, 6: 10, 7: 13, 8: 9, 9: 9}
         for k, v in centrality.items():
             self.assertAlmostEqual(v, expected[k])
+
+
+class TestDegreeCentrality(unittest.TestCase):
```

Comment:
The existing test coverage is good, but remember to add a test with the multigraph case to check that we cover it properly